### PR TITLE
docs: add hssi ethernet tools markdown

### PR DESCRIPTION
**Pull request by anandaravuri**
-add hssistats markdown
  -add hssiloopback markdown
  -add hssimac markdown

Signed-off-by: aravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+  Prints usage information
+
+`--pcie-address PCIE_ADDRESS, -P PCIE_ADDRESS`
+
+  The PCIe address of the desired fpga in ssss:bb:dd.f format. sbdf of device to program (e.g. 04:00.0 or 0000:04:00.0). Optional when one device in system.
+
+## EXAMPLES ##
+
+`hssistats --pcie-address  <0000:04:00.0`
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:24]: Remove the <

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  prints the MAC statistics 
+
+
+## SYNOPSIS ##
+```console
+hssiloopback [-h] [--pcie-address PCIE_ADDRESS, -P PCIE_ADDRESS] --loopback [{enable,disable}]
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:31]: Should probably put each doc in its own file.

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  The PCIe address of the desired fpga  in ssss:bb:dd.f format. sbdf of device to program (e.g. 04:00.0 or 0000:04:00.0). Optional when one device in system.
+
+
+`--loopback [{enable,disable}]`
+
+  Ethernet enable or disable loopback.
+  
+## EXAMPLES ##
+
+`hssiloopback --pcie-address  <0000:04:00.0` --loopback enable
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:55]: Remove the <

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+  Enables ethernet loopback
+ 
+ 
+ `hssiloopback --pcie-address  <0000:04:00.0` --loopback disable
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:60]: Remove the <

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+  Disable ethernet loopback
+
+
+## SYNOPSIS ##
+```console
+hssimac [-h] [--pcie-address PCIE_ADDRESS] [--mtu [MTU]]
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:67]: Should probably put each tool in its own file.

File: doc/src/fpga_tools/eth_tools/eth_tools.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  The PCIe address of the desired fpga  in ssss:bb:dd.f format. sbdf of device to program (e.g. 04:00.0 or 0000:04:00.0). Optional when one device in system.
+
+
+`--mtu [MTU]`
+
+  Maximum allowable ethernet frame length.
+
+## EXAMPLES ##
+
+`hssimac --pcie-address  <0000:04:00.0> --mtu`
```
Comment (tswhison) [inline doc/src/fpga_tools/eth_tools/eth_tools.md:91]: Remove the < and >

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 1 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: fixed

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 2 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: fixed

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 3 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: fixed

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 2 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: new markdown file hssilpbk.md

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 5 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: new markdown file hssimac.md

File: doc/src/fpga_tools/eth_tools/eth_tools.md (hunk 6 above)
Comment (anandaravuri) [reply doc/src/fpga_tools/eth_tools/eth_tools.md]: fixed

File: doc/sphinx/conf.py.in [hunk 7]
```diff
@@ -169,6 +169,9 @@ man_pages = [
     ("docs/fpga_tools/pci_device/pci_device", 'pci_device', u'Common operations for PCIe devices', [author], 8),
     ("docs/fpga_tools/opae.io/opae.io", 'opae.io', u'User space access to PCIe devices', [author], 8),
     ("docs/fpga_tools/host_exerciser/host_exerciser.md", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssistats.md", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssimac.md", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssilpbk.md", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
```
Comment (tswhison) [inline doc/sphinx/conf.py.in:174]: s,hssiloopback,hssilpbk, to match the markdown name.

Comment (anandaravuri) [reply doc/sphinx/conf.py.in]: fixed,  name as file name and script hssiloopback

**Review by tswhison — APPROVED** [review]

**Review by michelleyho — APPROVED** [review]

**Pull request merged**